diff --git a/firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+KeyCommands.swift b/firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+KeyCommands.swift
index 19d82dcfc1..6b72f98b4a 100644
--- a/firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+KeyCommands.swift
+++ b/firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+KeyCommands.swift
@@ -382,6 +382,11 @@ extension BrowserViewController {
             UIKeyCommand(action: #selector(selectTabSix), input: "6", modifierFlags: .command),
             UIKeyCommand(action: #selector(selectTabSeven), input: "7", modifierFlags: .command),
             UIKeyCommand(action: #selector(selectTabEight), input: "8", modifierFlags: .command),
+            UIKeyCommand(
+                action: #selector(toggleFloorpOverlayDrawerKeyCommand),
+                input: "d",
+                modifierFlags: [.command, .shift]
+            ),
         ] + windowShortcuts
 
         let isEditingText = tabManager.selectedTab?.isEditing ?? false
